docs/en/stack/ml/df-analytics/flightdata-classification.asciidoc

```diff
@@ -0,0 +1,477 @@
+exclude fields that either contain erroneous data or describe the 
+`dependent_variable`.
+<6> Specifies a memory limit for the job. If the job requires more than this 
+amount of memory, it fails to start. This makes it possible to prevent job 
+execution if the available memory on the node is limited.
+
+
+The API returns the following response:
+
+[source,console-result]
+--------------------------------------------------  
+{
+  "id" : "model-flight-delays",
+  "source" : {
+    "index" : [
+      "kibana_sample_data_flights"
+    ],
+    "query" : {
+      "match_all" : { }
+    }
+  },
+  "dest" : {
+    "index" : "df-flight-delayed",
+    "results_field" : "ml"
+  },
+  "analysis" : {
+    "regression" : {
+      "dependent_variable" : "FlightDelayMin",
+      "prediction_field_name" : "FlightDelayMin_prediction",
+      "training_percent" : 10.0
+    }
+  },
+  "analyzed_fields" : {
+    "includes" : [ ],
+    "excludes" : [
+      "Cancelled",
+      "FlightDelay",
+      "FlightDelayType"
+    ]
+  },
+  "model_memory_limit" : "100mb",
+  "create_time" : 1574436219144,
+  "version" : "7.5.0",
+  "allow_lazy_start" : false
+}
+--------------------------------------------------
+--
+
+. Start the job.
++
+--
+Use the {ref}/start-dfanalytics.html[start {dfanalytics-jobs}] API to start the 
+job. It will stop automatically when the analysis is complete, you don't need to 
+stop it manually.
+
+[source,console]
+--------------------------------------------------
+POST _ml/data_frame/analytics/model-delayed-flights/_start
+--------------------------------------------------
+// TEST[skip:TBD]
+
+
+The job takes a few minutes to run. Runtime depends on the local hardware and 
+also on the number of documents and fields that are analyzed. The more fields 
+and documents, the longer the job to run.
+--
+
+. Check the job stats to follow the progress by using the 
+{ref}/get-dfanalytics-stats.html[get {dfanalytics-jobs} statistics API].
++
+--
+
+[source,console]
+--------------------------------------------------
+GET _ml/data_frame/analytics/model-delayed-flights/_stats
+--------------------------------------------------
+// TEST[skip:TBD]
+
+
+The API call returns the following response: 
+
+[source,console-result]
+----  
+{
+  "count" : 1,
+  "data_frame_analytics" : [
+    {
+      "id" : "model-delayed-flights",
+      "state" : "stopped",
+      "progress" : [
+        {
+          "phase" : "reindexing",
+          "progress_percent" : 100
+        },
+        {
+          "phase" : "loading_data",
+          "progress_percent" : 100
+        },
+        {
+          "phase" : "analyzing",
+          "progress_percent" : 100
+        },
+        {
+          "phase" : "writing_results",
+          "progress_percent" : 100
+        }
+      ]
+    }
+  ]
+}
+----  
+
+The job has four phases. When all the phases have completed, the job 
+state becomes `stopped` and the results are ready to view and evaluate.
```

Comment:
This made me wonder whether there might be other reasons why the state would be "stopped" (e.g. if you stop it manually before it's complete), so I suggest omitting that detail:
```suggestion
stops and the results are ready to view and evaluate.
```